thirdparty/tinkerpop/src/test/java/org/calrissian/accumulorecipes/thirdparty/tinkerpop/BlueprintsGraphStoreTest.java

```diff
@@ -113,7 +120,7 @@ public void testVertices() {
     }
 
     @Test
-    public void testEdges() {
+    public void testEdges() throws TableNotFoundException {
```

Comment:
First off this is not a bug and I am not suggesting changing it.  Just a tip.

So it is just something that I do ONLY for tests.  The biggest reason is that in tests, unlike the main source code you should always throw exceptions instead of handling them so that the test would fail properly.  Over time as code gets refactored and dependencies change it can be a hassle to fix all the throws statements in tests.  As a result I have just noticed it is easier in the long run to just use Exception for tests.